src/validate.js

```diff
@@ -41,7 +41,7 @@ var browsersSchema = exports.browsersSchema = Joi.array().min(1).unique().items(
   }),
   Joi.object().keys({
     browserName: Joi.string().valid('chrome', 'firefox', 'safari', 'microsoftedge', 'internet explorer').required(),
-    version: Joi.string().regex(/^\d+\.\d+$/).required(),
+    version: Joi.string().regex(/^(\d+)(\.\d+)?(\.\d+)?$/).required(),
```

Comment:
Please remove `X` format. Versions are used for Sauce browsers only, and should include at least major+minor